### PR TITLE
Reservoir Cold Start Water Elevation Check

### PR DESCRIPTION
**Pull request by jdmattern-noaa**
Change initial reservoir cold start water elevation to -1e9 and check to less than -900000000 to allow negative water elevations at warm start.

## Additions

-

## Removals

-

## Changes

-

## Testing

1.

## Notes

-

## Todos

-

## Checklist

- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Code follows project standards (link if applicable)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Target Environment support

- [ ] Windows
- [ ] Linux
- [ ] Browser

### Accessibility

- [ ] Keyboard friendly
- [ ] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Flexible from small to large screens
- [ ] No linting errors or warnings
- [ ] JavaScript tests are passing

### CODE REVIEW

File: src/nwm_routing/src/nwm_routing/__main__.py [hunk 1]
```diff
@@ -431,7 +431,7 @@ def main_v02(argv):
         else:
             # TODO: Consider adding option to read cold state from route-link file
             waterbodies_initial_ds_flow_const = 0.0
```
Comment (jdmattern-noaa) [inline src/nwm_routing/src/nwm_routing/__main__.py:433]: @jameshalgren Do we need to put this same "if restart_parameters.get" check in main_v03?

Comment (jameshalgren) [reply src/nwm_routing/src/nwm_routing/__main__.py:433]: We do. I think that would go into preprocess.py.

Comment (jdmattern-noaa) [reply src/nwm_routing/src/nwm_routing/__main__.py:433]: Updated in preprocess.py

**Review by awlostowski-noaa — APPROVED** [review]
Thanks, David!

**Pull request merged**